crates/daphne-worker/src/durable/test_state_cleaner.rs

```diff
@@ -43,14 +43,6 @@ impl TestStateCleaner {
             Some(bindings::TestStateCleaner::Put) => {
                 let durable_ref: DurableReference =
                     serde_json::from_slice(&req.bytes().await?).unwrap();
-                match durable_ref.binding.as_ref() {
-                    bindings::AggregateStore::BINDING => (),
-                    s => {
-                        let message = format!("GarbageCollector: unrecognized binding: {s}");
-                        console_error!("{}", message);
-                        return Err(int_err(message));
-                    }
-                };
 
```

Comment:
Why delete this?

Reply:
The durable objects runtime already tells us if a durable object binding doesn't exist when we try to use it, so why check ourselves? This just makes it so we have to manually update this match statement every time we add a DO